packages/collections/pub/controlcard.php

```diff
@@ -39,6 +42,17 @@
             if($objFile->FileType->MediaType->MediaType == 'Image')
             {
                $containsImages = true;
+                $img = '<div class="thumbnailimg">
+                            <div class="thumbnailimgwrapper">
+                               <a class="thumbimglink" href="?p=digitallibrary/digitalcontent&amp;id='.$objFile->DigitalContentID.'" title="'.$objDigitalContent->getString('Title', 30).'" rel="#mediumPreview">
+                                  <img class="digcontentfile" src="'.$objFile->getFileURL(DIGITALLIBRARY_FILE_PREVIEWSHORT).'" alt="'.$objFile->getString('Title').'"/>
+                               </a>
+                            </div>'.
+//                            <div class="thumbnailcaption">
+//                               <a href="?p=digitallibrary/digitalcontent&amp;id='.$objFile->DigitalContentID.'">'.$objDigitalContent->getString('Title', 30).'</a>
+//                            </div>

```

Comment:
We will be using the caption, I just don't know what data will link to it at the moment.